config/install/field.storage.paragraph.field_paragraph_search_block.yml

```diff
@@ -0,0 +1,29 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - options
+    - paragraphs
+id: paragraph.field_paragraph_search_block
+field_name: field_paragraph_search_block
+entity_type: paragraph
+type: list_string
+settings:
+  allowed_values:
+    -
+      value: none
+      label: None
+    -
+      value: events
```

Comment:
Do you think the value should match the content type's machine name? i.e `event` instead of `events`. There's no use case for it presently but I think in the future if we want to programmatically consume that value it'd be better if it matches the machine name.

This was the approach taken [for Card Carousel](https://github.com/dpc-sdp/content-vic-gov-au/pull/283/files#diff-3159ab616ed5a32fddea76de360f9682R15).

Reply:
Understood , recommendation will be implemented